packages/manager/modules/pci/src/components/project/storages/databases/flavor.class.js

```diff
@@ -18,6 +17,14 @@ export default class Flavor {
     return some(this.availability, { network: networkName });
   }
 
+  get minDiskSize() {
+    return get(head(this.availability), 'minDiskSize');
+  }
+
+  get maxDiskSize() {
+    return get(head(this.availability), 'maxDiskSize');
```

Comment:
i think we can avoid to use Lodash get and head functions.

```suggestion
    return (this.availability || [])[0]?.maxDiskSize;
```

Reply:
Done here: https://github.com/ovh/manager/pull/5932/commits/ee7b1386639352dac98ee930dcb1e2b39891123c